actions/promote/promote.js

```diff
@@ -17,16 +17,18 @@
 
 // eslint-disable-next-line import/no-extraneous-dependencies
 const openwhisk = require('openwhisk');
-const { getAioLogger } = require('../utils');
+const { getAioLogger, updateStatus } = require('../utils');
 
 // This returns the activation ID of the action that it called
 function main(args) {
     const logger = getAioLogger();
     let payload;
+    const { spToken, adminPageUri, projectExcelPath, projectRoot } = args;
+    const projectPath = `${projectRoot}${projectExcelPath}`;
```

Comment:
We might need a separate check for missing `projectRoot` as mentioned for the copy worker.